app/package.json

```diff
@@ -83,18 +85,17 @@
     "web3-react": "^5.0.4"
   },
   "scripts": {
-    "build": "sed -i '207s/isEnvProduction/false/' node_modules/react-scripts/config/webpack.config.js && GENERATE_SOURCEMAP=false node --max-old-space-size=4000 node_modules/react-scripts/scripts/build.js",
-    "eject": "react-scripts eject",
+    "start": "npx react-scritps start",
```

Comment:
This has a typo.

Reply:
Besides, does it make sense to use `react-scripts` after an eject? I thought that you are supposed to do `node scripts/whatever.js`.

Reply:
@fvictorio where is the typo? :eyes: 

Good question! I tried! but I had to add a looooot of dependencies and I ran into some problems with the paths. So I only added the changes to make the build with scripts and custom config, and I left the rest to be done in a next PR. But yeah, we will want to move to scripts eventually :)

Reply:
found the typo, fixed.